ui/page/tickets/overview.go

```diff
@@ -52,7 +51,7 @@ func NewTicketPage(l *load.Load) *Page {
 		ticketPageContainer: &layout.List{Axis: layout.Vertical},
 		purchaseTicket:      l.Theme.Button(new(widget.Clickable), "Purchase"),
 
-		autoPurchaseEnabled: new(widget.Bool),
+		autoPurchaseEnabled: l.Theme.Switch(),
```

Comment:
the `autoPurchaseEnabled` feature seem not to have been implemented. Kindly disable the button toggle.
`autoPurchaseEnabled.Disabled()`

Reply:
Would be great if create new PR for that function

Reply:
Switch widget disabled() already exist

Reply:
> the `autoPurchaseEnabled` feature seem not to have been implemented. Kindly disable the button toggle.
> `pg.autoPurchaseEnabled.Disabled()` in the `OnResume()` method

<img width="566" alt="image" src="https://user-images.githubusercontent.com/27733432/131457788-3af07dc2-9ff8-4c1e-b7c4-f450705cf731.png">

What was requested was to have the auto purchase toggle disabled not removed.
Kindly add it back as it is a vital part of the UI... all you need to so is the following.

`func (pg *Page) OnResume() {
	pg.ticketPrice = dcrutil.Amount(pg.WL.TicketPrice()).String()
	go pg.WL.GetVSPList()

         // TODO pruchase ticket functionality
        pg.autoPurchaseEnabled.Disabled()
}`